Gemfile.lock

```diff
@@ -291,4 +310,4 @@ RUBY VERSION
    ruby 2.5.1p57
 
 BUNDLED WITH
-   1.16.5
+   1.17.1
```

Comment:
Also ran `bundle update`, not sure if this is a problem / potential breaking change @ethanlee16